src/webui/api/appcontroller.cpp

```diff
@@ -235,8 +235,8 @@ void AppController::preferencesAction()
     data["web_ui_port"] = pref->getWebUiPort();
     data["web_ui_upnp"] = pref->useUPnPForWebUIPort();
     data["use_https"] = pref->isWebUiHttpsEnabled();
-    data["web_ui_https_cert_path"] = pref->getWebUIHttpsCertificatePath();
-    data["web_ui_https_key_path"] = pref->getWebUIHttpsKeyPath();
+    data["web_ui_https_cert_path"] = pref->getWebUIHttpsCertificatePath().data();
```

Comment:
> I think those should be `toString()` (and other `data()` instances in this function).

Initially I returned "native" paths only where `toNative()` was used explicitly. Should I change it in entire API to return "native" paths?
Is it generally a good idea to make API results dependent on the server platform? I would use uniform paths instead.

Reply:
>Is it generally a good idea to make API results dependent on the server platform?

In this case I think so, because it is referring to some file on the server.

>Should I change it in entire API to return "native" paths?

For this PR it should just preserve the original outcome. So maybe later.